index.html

```diff
@@ -703,12 +712,52 @@ <h2>The <dfn>ExposedThing</dfn> interface</h2>
       <p>
         Start serving external requests for the <a>Thing</a>, so that WoT interactions using <a>Properties</a>, <a>Action</a>s and <a>Event</a>s will be possible.
       </p>
+      <p>
+        The <code>expose()</code> method MUST run the following steps:
+        <ol>
+          <li>
+            Return a <code>Promise</code> <var>promise</var> and execute the next steps <a>in parallel</a>.
+          </li>
+          <li>
+            If invoking <code>expose()</code> is not allowed for the current scripting context for security reasons, reject <var>promise</var> with <code>SecurityError</code> and terminate these steps.
+          </li>
+          <li>
+            Make a request to the underlying platform to create and configure serving external requests for WoT interactions, based on the <a>Protocol Bindings</a>.
```

Comment:
"create and configure serving external requests" -- in particular "create" sounds confusing.
maybe: """
Make a request to the underlying platform to attach protocol listeners to the WoT interactions, based on the <a>Protocol Bindings</a>.
"""